python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -255,7 +255,7 @@ def determinePsf(
         wcs = {0: galsim.PixelScale(1.0)}
         pointing = None
 
-        logger = logging.getLogger(self.log.getName()+".Piff")
+        logger = logging.getLogger(self.log.name + ".Piff")
```

Comment:
This one should be written as something like 

```python
logger = self.log.getChild("Piff")
```
I'm a bit surprised that this is needed since doesn't it imply that this Task is not being given a piff specific name itself?

Reply:
I'm not sure...I was just trying to fix up the `log.getName()` calls.  I'll again ask if @jmeyers314 has any comments.

Reply:
The line as written should definitely use getChild.  There is no debate about that. The debate is over why this task seemingly has an incorrect logger name to start with.